Various updates (#2617)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/hybrid-deployment/set-up-microsoft-365-groups.md

Microsoft 365 Groups service that enables teams to communicate, schedule meetings, and collaborate on documents more easily. All information shared with a group, from email messages sent to the group, to files stored in the group's OneDrive for Business or SharePoint libraries, is available to any member of a group. If you've configured a hybrid deployment between your on-premises Exchange organization and Microsoft 365 or Office 365, you can make groups created in Microsoft 365 or Office 365 available to your on-premises users by following the steps in this topic.

> Following are experiences that On-Premises mailbox can expect after all the steps in this document have been performed:

Reviewer: ```suggestion
The following are experiences that on-premises mailbox users can expect after all the steps in this document have been performed:
```

> 1) Expand Microsoft 365 Group and view members, just like a distribution group, in Outlook/OWA client while composing new email

Reviewer: ```suggestion
- Expand a Microsoft 365 group and view the members, similar to a distribution group, in Outlook on the web and Outlook desktop client while composing a new email.
```

> 2) Send/receive emails, calendar invites to Microsoft 365 group 

Reviewer: ```suggestion
- Send and receive emails and calendar invites to and from Microsoft 365 groups.
```

> 3) Collaborate on ODB documents sent by Microsoft 365 Users

Reviewer: ```suggestion
- Collaborate on ODB documents sent by Microsoft 365 users.
```

> 4) Access SharePoint site, Planner, OneNote associated with Microsoft 365 Group

Reviewer: ```suggestion
- Access a SharePoint site, Planner, and OneNote associated with the Microsoft 365 group.
```

> [!IMPORTANT]

> After the steps in this article have been successfully performed, the on-premises mailbox can perform above mentioned tasks. However, the Microsoft 365 group will not appear in the navigation bar of Outlook on the web or Outlook desktop client. For a full-fledged Microsoft 365 Groups experience, the mailbox must be present in Microsoft 365. 

Reviewer: ```suggestion
> After the steps in this article have been successfully performed, the on-premises mailbox can perform the previously mentioned tasks. However, the Microsoft 365 group will not appear in the navigation bar of Outlook on the web or Outlook desktop client. For a full-fledged Microsoft 365 Groups experience, the mailbox must be present in Microsoft 365. 
```

## Enable Group writeback in Azure AD Connect

> This step synchronizes Microsoft 365 Group from Exchange Online to Exchange On-Premises.

Reviewer: ```suggestion
This step synchronizes Microsoft 365 Groups from Exchange Online to Exchange on-premises.
```

Author: The line I mentioned is correct since its one direction sync, i.e. from EXO To On-Premises. If we mention "with" it sounds like changes made to Group at On-Premises will be synced to EXO, which is not the case

Reviewer: @Batre-MSFT - Agreed. I've changed the suggestion accordingly. Could you please commit the other suggestions?

## Configure a group domain

> The primary SMTP domain of a Microsoft 365 group is called a group domain. By default, the default accepted domain in your Exhange Online organization is chosen as the group domain. For better interoperation with On-Premises servers, it is recommended to add a dedicated groups domain. You can add a domain using the following steps. For more information about multi-domain support for Microsoft 365 Groups, check out [Multi-domain support for Microsoft 365 Groups](/microsoft-365/admin/create-groups/choose-domain-to-create-groups).

Reviewer: ```suggestion
The primary SMTP domain of a Microsoft 365 group is called a group domain. By default, the default accepted domain in your Exchange Online organization is chosen as the group domain. For better interoperation with on-premises servers, we recommended adding a dedicated groups domain. You can add a domain using the following steps. For more information about multi-domain support for Microsoft 365 Groups, check out [Multi-domain support for Microsoft 365 Groups](/microsoft-365/admin/create-groups/choose-domain-to-create-groups).
```

   > [!CAUTION]
   > If the MX DNS record for the group domain is set to the on-premises Exchange server, mail flow won't work correctly between users in the on-premises Exchange organization and the Microsoft 365 group.

3. Add the group domain as an accepted domain in your on-premises Exchange organization using the following command. This is needed so that the hybrid Send connector can be used to deliver outbound mail to the group domain in Microsoft 365 or Office 365.

   ```PowerShell
   New-AcceptedDomain -Name groups.contoso.com -DomainName groups.contoso.com -DomainType InternalRelay
   ```

> [!IMPORTANT]

> Ensure On-Premises servers can resolve the MX entry added for the groups domain in step 2. The groups domain must be added as InternalRelay, otherwise it will cause mailflow issues.

Reviewer: ```suggestion
> Ensure that on-premises servers can resolve the MX entry added for the group's domain in step 2. The group's domain must be added as InternalRelay, otherwise, it will cause mail flow issues.
```

### Test using an on-premises mailbox

> 1. Add an on-premises mailbox to a Microsoft 365 group

Reviewer: ```suggestion
1. Add an on-premises mailbox to a Microsoft 365 group.
```

  Update-Recipient -Identity "[group Distinguished Name]"
  ```

> - If the "Oubound to Office 365" Send-Connector is using an Edge Transport Server as the source server. Messages to Groups will end-up in a loop and NDR. Please check (this)[https://docs.microsoft.com/en-us/Exchange/mail-flow/accepted-domains/accepted-domains?view=exchserver-2019] article for more details

Reviewer: ```suggestion
- If the _Outbound to Office 365_ Send connector is using an Edge transport server as the source server, messages to Microsoft 365 Groups will end up in a loop, resulting in a non-delivery report. Check out [Accepted domains in Exchange Server](/Exchange/mail-flow/accepted-domains/accepted-domains?view=exchserver-2019) for more details.
```

- **Mail sent to a group's secondary SMTP address fails to be delivered**: When multiple email addresses are added to a group, only the primary SMTP address is written back to your on-premises Active Directory. If an on-premises user tries to send a message to the secondary SMTP address of a group, the message will fail to be delivered. To prevent this issue, configure only one SMTP address on each group.

- **Delivery of external mail to a group fails if you've enabled centralized mail flow**: If centralized mail flow is enabled, mail sent by an external user to a group fails to be delivered, even though the group allows email from external senders.

- **On-premises users can't send mail as a group**: An on-premises user who tries to send a message as a Microsoft 365 group will receive a permission denied error even if they're given Send As permissions on the group. Send As permissions on a group work only for Exchange Online mailbox users.

> - **By default, when On-Premises mailbox sends an email message to an Outlook group that user a member of, the user doesn't receive a copy of that message in Inbox.**

Reviewer: ```suggestion
- **By default, when an email is sent from an on-premises mailbox to an Outlook group that the user is a member of, the user doesn't receive a copy of that email in their Inbox**: The Exchange Tenant Admin can use the following Exchange Online shell command to ensure the on-premises mailbox user can receive a copy of the email in their Inbox:
```

> EXO Tenan Admin can use following EXO shell command to ensure On-Premises mailbox can receive copy of email in Inbox, for above mentioned scenario:

Reviewer: ```suggestion
```

Author: There's a typo, it should be EXO Tenant Admin

Reviewer: Fair enough, but if you commit this suggestion, it will simply delete the line, because the information is now included in line 169. Could you please commit this suggestion?

  ```PowerShell
  
  Get-MailUser <MEU for On-Premises mailbox> | Set-MailboxMessageConfiguration -EchoGroupMessageBackToSubscribedSender $true
  
  ```
- **On-premises users can't send mail as a group**: An on-premises user who tries to send a message as a Microsoft 365 group will receive a permission denied error even if they're given Send As permissions on the group. Send As permissions on a group work only for Exchange Online mailbox users.

> - **Allowing On-premises users to manage Microsoft 365 group**: On-premises users can be assigned as owner of Microsoft 365 Group. However, the On-Premises users will have to use (Azure AD Web Portal)[https://account.activedirectory.windowsazure.com/r#/groups] to manage the groups they own. The Azure AD admin must enable self-service management in AAD portal using steps provided (here)[https://docs.microsoft.com/en-us/azure/active-directory/enterprise-users/groups-self-service-management]

Reviewer: ```suggestion
- **Allowing on-premises users to manage a Microsoft 365 group**: On-premises users can be assigned as owners of a Microsoft 365 group. However, the on-premises users will have to use the (Azure AD web portal)[https://account.activedirectory.windowsazure.com/r#/groups] to manage the groups they own. The Azure AD admin must enable self-service management in the Azure AD portal using steps provided in [Set up self-service group management in Azure Active Directory](/azure/active-directory/enterprise-users/groups-self-service-management).
```

> - **Selecting a group from Outlook's left navigation pane doesn't open the group's mailbox for Exchange Online user**: Outlook uses the AutoDiscover URL to open a group mailbox. If a group's primary email address is in a domain that doesn't point to the Microsoft 365 or Office 365 AutoDiscover URL (autodiscover.outlook.com), Outlook won't be able to open the group's mailbox. To fix the issue, groups can be provisioned with a primary address in a domain that points to the Microsoft 365 or Office 365 AutoDiscover URL. You can configure an email address policy to add a primary email address on each group mailbox that points to that AutoDiscover URL. Check out [Choose the domain to use when creating Microsoft 365 groups](/microsoft-365/admin/create-groups/choose-domain-to-create-groups) for more details.

Reviewer: ```suggestion
- **Selecting a group from Outlook's left navigation pane doesn't open the group's mailbox for an Exchange Online user**: Outlook uses the AutoDiscover URL to open a group mailbox. If a group's primary email address is in a domain that doesn't point to the Microsoft 365 or Office 365 AutoDiscover URL (autodiscover.outlook.com), Outlook won't be able to open the group's mailbox. To fix this issue, groups can be provisioned with a primary address in a domain that points to the Microsoft 365 or Office 365 AutoDiscover URL. You can configure an email address policy to add a primary email address on each group mailbox that points to that AutoDiscover URL. Check out [Choose the domain to use when creating Microsoft 365 groups](/microsoft-365/admin/create-groups/choose-domain-to-create-groups) for more details.
```
